deployment-scanner/src/main/java/org/jboss/as/server/deployment/scanner/FileSystemDeploymentService.java

```diff
@@ -590,6 +612,27 @@ private void executeScannerTasks(List<ScannerTask> scannerTasks, DeploymentOpera
         }
     }
 
+    private class ManualScanCallable implements Callable<List<?>> {
```

Comment:
The singleScan method can be a void return now, so there's no issue re: ScannerTask becoming API.